File: todo-frontend/application/lib/model/login/login_response.dart
class LoginResponse {
  final String refreshToken;
  final String accessToken;
  final String fullName;

  LoginResponse({
    required this.accessToken,
    required this.refreshToken,
    required this.fullName
  });

  factory LoginResponse.fromJson(Map<String, dynamic> json) {
    return LoginResponse(
      accessToken: json["accessToken"] ?? "",
      refreshToken: json["refreshToken"] ?? "",
      fullName: json["fullName"] ?? ""
      );
  }
}

File: todo-backend/server-todo/src/main/java/com/example/servertodo/dto/auth/login/LoginResponse.java
package com.example.servertodo.dto.auth.login;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;

@Getter
@AllArgsConstructor
@Builder
public class LoginResponse {
    private String fullName;
    private String accessToken;
    private String refreshToken;
}
